docs/hoon/twig/wut-test/hep-case.md

```diff
@@ -11,13 +11,23 @@ union, with no default.
 ## Expands to
 
 ```
+:loop
 :ifno  q
   :lost 
 :if  :fits(p.i.q p)
   q.i.q
 :moar(q t.q)
 ```
 
+```
+|-
+?.  q
+  :lost
+?:  ?=(p.i.q p)
+  q.i.q
+$(q t.q)
+```
+
 ## Syntax
```

Comment:
... ;)

I guess it depends on the purpose of the expansions; `:lost` is a syntax error, so it seems like a cruel kind of nerd-sniping to include it in the official docs. Does the distinction between it and a tracing printf matter to anyone who won't be reading the compiler source?